### PR TITLE
MAUI-4945 Updated the UG documentation content for Customization APIs and Text property and Completed event of SfAutoComplete and SfComboBox.

### PR DESCRIPTION
**Pull request by BrundhaVelusamy**
Prepared UG for .NET MAUI SfAutoComplete and SfComboBox controls for ItemTemplate customization support,  Text property, and Completed event.

### CODE REVIEW

File: MAUI/Autocomplete/Getting-Started.md [hunk 1]
```diff
@@ -238,4 +238,8 @@ The following gif image illustrates the result of the above code:
 
 You can find the complete getting started sample of .NET MAUI Autocomplete from this [link.](https://github.com/SyncfusionExamples/maui-autocomplete-samples)
 
+## Text
 
+The [Text]() property used to gets the user-submitted text in the [SfAutoComplete](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Inputs.SfAutocomplete.html). The default value of the `Text` property is `string.Empty`.
```
Comment (florenceliliana) [inline MAUI/Autocomplete/Getting-Started.md:243]: The [Text]() property is used to get the user-submitted text in the [SfAutoComplete](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Inputs.SfAutocomplete.html). The default value of the `Text` property is the `string.Empty`.

File: MAUI/ComboBox/Getting-Started.md [hunk 2]
```diff
@@ -246,3 +246,9 @@ The following gif image illustrates the result of the above code:
 ![.NET MAUI ComboBox choose item using editing](Images/GettingStarted/EditableMode.png)
 
 You can find the complete getting started sample of .NET MAUI ComboBox from this [link.](https://github.com/SyncfusionExamples/maui-combobox-samples)
+
+## Text
+
+The [Text]() property used to gets the user-submitted text in the [SfComboBox](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Inputs.SfComboBox.html) editable mode. The default value of the `Text` property is `string.Empty`.
```
Comment (florenceliliana) [inline MAUI/ComboBox/Getting-Started.md:252]: The [Text]() property used to get the user-submitted text in the [SfComboBox](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Inputs.SfComboBox.html) editable mode. The default value of the `Text` property is the `string.Empty`.

File: MAUI/ComboBox/UI-Customization.md [hunk 3]
```diff
@@ -173,4 +173,528 @@ comboBox.MaxDropDownHeight = 150;
...
+{% endhighlight %}
+{% endtabs %}
+
+The following image illustrates the result of the above code:
+
+![.NET MAUI ComboBox ItemTemplate](Images/UICustomization/ItemTemplate.png)
+
+## Customize the DropDown (suggestion) item based on condition
+
+The [ItemTemplate]() property helps you to decorate drop-down items conditionally based on their content using the custom templates. The default value of `ItemTemplate` is `null`.
```
Comment (florenceliliana) [inline MAUI/ComboBox/UI-Customization.md:347]: The [ItemTemplate]() property helps you to decorate drop-down items conditionally based on their content using the custom templates. The default value of the `ItemTemplate` is `null.`

File: MAUI/ComboBox/UI-Customization.md [hunk 4]
```diff
@@ -173,4 +173,528 @@ comboBox.MaxDropDownHeight = 150;
...
+{% endhighlight %}
+{% endtabs %}
+
+The following image illustrates the result of the above code:
+
+![.NET MAUI ComboBox ItemTemplateSelector](Images/UICustomization/TemplateSelector.png)
+
+## Completed Event
+
+The [Completed]() event is raised when the user finalizes the text in the [SfComboBox](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Inputs.SfComboBox.html) editable mode by pressing return key on the keyboard.The handler for the event is a generic event handler, taking the `sender` and `EventArgs`(the `EventArgs` value is `string.Empty`):
```
Comment (florenceliliana) [inline MAUI/ComboBox/UI-Customization.md:666]: The [Completed]() event is raised when the user finalizes the text in the [SfComboBox](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Inputs.SfComboBox.html) editable mode by pressing return key on the keyboard. The handler for the event is a generic event handler, taking the `sender` and `EventArgs`(the `EventArgs` value is `string.Empty`):

**Review by florenceliliana — CHANGES_REQUESTED** [review]

**Review by PaulAndersonS — APPROVED** [review]

**Pull request merged**